{
  "uid" : "b15e0d0f32f3bf2e",
  "name" : "test_72[data0]",
  "fullName" : "test_13.TestSystem#test_72",
  "historyId" : "2cf99a46307f0743b5ebb9c38ed97fd9",
  "time" : {
    "start" : 1602840914805,
    "stop" : 1602840914805,
    "duration" : 0
  },
  "status" : "skipped",
  "statusMessage" : "Skipped: 默认仓库盘点入库，未知异常BUG！！",
  "statusTrace" : "('D:\\\\seleniumweb\\\\saas\\\\TestCase1\\\\test_13.py', 43, 'Skipped: 默认仓库盘点入库，未知异常BUG！！')",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "dealer_session_login",
    "time" : {
      "start" : 1602840879572,
      "stop" : 1602840911634,
      "duration" : 32062
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \n\n",
    "statusTrace" : "  File \"d:\\seleniumweb\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"d:\\seleniumweb\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"d:\\seleniumweb\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 908, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\seleniumweb\\saas\\TestCase1\\conftest.py\", line 23, in dealer_session_login\n    loc(driver).get_login(LD.login_data[\"username\"], LD.login_data[\"password\"], auth)\n  File \"D:\\seleniumweb\\saas\\Locators\\CommonLT.py\", line 37, in get_login\n    self.click_element(self.click_subscribe)\n  File \"D:\\seleniumweb\\saas\\Common\\basepage.py\", line 46, in click_element\n    self.get_element(EC.element_to_be_clickable, locator).click()\n  File \"D:\\seleniumweb\\saas\\Common\\basepage.py\", line 29, in get_element\n    self.wait_vs(ec, locator)\n  File \"D:\\seleniumweb\\saas\\Common\\basepage.py\", line 18, in wait_vs\n    WebDriverWait(self.driver, timeout=10, poll_frequency=0.5).until(ec(locator))\n  File \"d:\\seleniumweb\\python\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "stepsCount" : 0,
    "hasContent" : true
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "story",
    "value" : "盘点"
  }, {
    "name" : "tag",
    "value" : "smoke"
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.skip('默认仓库盘点入库，未知异常BUG！！')"
  }, {
    "name" : "suite",
    "value" : "test_13"
  }, {
    "name" : "subSuite",
    "value" : "TestSystem"
  }, {
    "name" : "host",
    "value" : "DESKTOP-BQG3M1K"
  }, {
    "name" : "thread",
    "value" : "22136-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_13"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "data",
    "value" : "{'add_name': '默认仓库'}"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ "@pytest.mark.skip('默认仓库盘点入库，未知异常BUG！！')", "smoke" ]
  },
  "source" : "b15e0d0f32f3bf2e.json",
  "parameterValues" : [ "{'add_name': '默认仓库'}" ]
}